package.json

```diff
@@ -10,6 +10,7 @@
     "compile": "lerna exec -- tsc --project tsconfig.cjs.json",
     "compile:esm": "lerna exec --no-private -- tsc --project tsconfig.esm.json",
     "format": "prettier --write '**/*.{js,json,md,ts,tsx,yml}'",
+    "postinstall": "ts-node node_modules/@rcgen/cli/bin/run --verbose --force",
```

Comment:
You could evaluate using `ts-node/register` or even something like [`interpret`](https://www.npmjs.com/package/interpret) to support typescript configs in general...

Reply:
since this is only a temporary solution, which will disappear again soon, but works now, i don't want to put much effort into it